hyperon_das/index.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass
+class IndexField:
```

Comment:
Is this class supposed to represent an Index or a Field? The way it's named it says it's representing a Field. I supposed it should represent an Index so the proper name would be "FieldIndex".